### ISSUE TITLE
module has no attribute error on 'attrs' module.

### ISSUE THREAD

**Issue by chrisbeardy** [python/mypy#10084]
I have the following in an attrs decorated class:

```python
import attr

@attr.s
class test:
    test_mode: Union[str, int] = attr.ib(
        init=False,
        default=0,
        converter=test_mode_to_string,
        on_setattr=attr.setters.convert,
    )
```

mypy is giving me the following error:

![image](https://user-images.githubusercontent.com/20585410/107667931-ffa01c80-6c87-11eb-9992-3a724622d7f9.png)

I am not expecting this to be an error as the `attr.setters.convert` does exist.

- Mypy version used: 0.800
- Mypy configuration options from `mypy.ini` (and other config files):
- mypi.ini
```
check_untyped_defs = True
pretty = True
show_column_numbers = True
show_error_codes = True
show_error_context = True
strict_equality = True
warn_redundant_casts = True
warn_return_any = True
warn_unreachable = True
warn_unused_configs = True
warn_unused_ignores = True
allow_untyped_globals = True
```
- Python version used: 3.6
- Operating system and version:  Windows 10

Using attrs: 20.3.0

**Comment by JelleZijlstra** [comment]
I  believe attrs bundles its  own stubs, so this issue should be reported  to them directly.

**Comment by chrisbeardy** [comment]
attrs seems to have the following stub file set up caller setters.pyi in which convert is defined:

```python
from typing import Any, NewType, NoReturn, TypeVar, cast

from . import Attribute, _OnSetAttrType


_T = TypeVar("_T")

def frozen(
    instance: Any, attribute: Attribute, new_value: Any
) -> NoReturn: ...
def pipe(*setters: _OnSetAttrType) -> _OnSetAttrType: ...
def validate(instance: Any, attribute: Attribute[_T], new_value: _T) -> _T: ...

# convert is allowed to return Any, because they can be chained using pipe.
def convert(
    instance: Any, attribute: Attribute[Any], new_value: Any
) -> Any: ...

_NoOpType = NewType("_NoOpType", object)
NO_OP: _NoOpType
```

**Comment by chrisbeardy** [comment]
I have got to the bottom of this, I realise now that as I am running mypy from a isolated virtualenv, I had not installed attrs into that venv, therefore was using the stubs still bundled with mypy which are out of date. This was my mistake and after installed attrs into the mypy venv, these errors are not shown.
